internal/cluster/cluster.go

```diff
@@ -51,7 +51,6 @@ var S3FileNames = []string{
 	"kubeadmin-password",
 	"kubeconfig-noingress",
 	"install-config.yaml",
-	"discovery.ign",
```

Comment:
Is there an enum in the swagger somewhere for this as well?

Reply:
there is. But we still want requests to `/clusters/$CLUSTER_ID/downloads/files?file_name=discovery.ign` to work...right?

Reply:
Yup, right, my bad :sweat_smile: